src/services/image.service.ts

```diff
@@ -0,0 +1,34 @@
+import type { Playlist, PlaylistItem } from '#types/playlist';
```

Comment:
Is this really a service?  It doesn't seem to access any external resources, which the other services do, though I suppose it's coupled to the JWP alternate images API convention...

Reply:
We could consider moving the logic to a custom hook or util and move the `generateAlternateImageUrl` to the API service, what do you think?

Reply:
Without actually digging into it, I would say probably it makes sense in the API service, since it's JW delivery API specific.